tiledb/sm/tile/tile.cc

```diff
@@ -39,41 +39,109 @@
 namespace tiledb {
 namespace sm {
 
+/* ****************************** */
+/*           STATIC API           */
+/* ****************************** */
+
+Status Tile::compute_chunk_size(
```

Comment:
- This is a mild refactor to replace some of the logic in `FilterPipeline:: compute_tile_chunks`. This was done because the chunk size computation now happens higher up the stack. So the logic in this implementation is a "given" for this patch.
- Chunk sizes are dependent a few different things, the implementation of this routine will explain it better. It does not default to a statically defined constant if that's what you're asking.

Reply:
I think I recall now, thanks for clarifying.